src/editor/serialize.js

```diff
@@ -33,10 +33,10 @@ export function mdSerialize(model) {
     }, "");
 }
 
-export function htmlSerializeIfNeeded(model) {
+export function htmlSerializeIfNeeded(model, forceHtml = false) {
```

Comment:
How about using an options object instead, so that callers are forced to name the boolean option? In this PR, you've done a similar thing with a commented boolean value, but by using an options object, it becomes mandatory to name the option at the call site.

```suggestion
export function htmlSerializeIfNeeded(model, { forceHtml = false }) {
```